vue/components/ui/textarea-platforme/textarea-platforme.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+    <div class="textarea-container">
+        <label-platforme
+            class="label description-label"
+            v-bind:text="descriptionLabel"
+            v-bind:for="id"
+            v-if="descriptionLabel"
+        />
+        <textarea
+            class="textarea"
+            v-bind:style="textAreaStyle"
+            v-bind:value="value"
+            v-bind:placeholder="placeholder"
+            v-bind:disabled="disabled"
+            v-bind:id="id"
+            ref="textarea"
+            v-on:input="onInput($event.target.value)"
+        />
+        <label-platforme
+            class="label help-label"
+            v-bind:size="'small'"
+            v-bind:text="helpLabel"
+            v-bind:for="id"
+            v-if="helpLabel"
+        />
+    </div>
+</template>
+
+<style lang="scss" scoped>
+@import "css/variables.scss";
+
+.textarea-container .label {
+    display: block;
+}
+
+.textarea-container .textarea {
+    background-color: $textarea-background-color;
+    border: solid 1px $textarea-border-color;
+    border-radius: 6px;
+    color: $grey;
+    display: block;
+    font-family: $font-family;
+    font-size: 14px;
+    font-weight: 600;
+    letter-spacing: 0.3px;
+    margin: 1px;
+    outline: none;
+    padding: 8px 8px;
+}
+
+.textarea-container .textarea::placeholder {
+    color: $light-grey;
+    font-family: $font-family;
+}
+
+.textarea-container .textarea:disabled,
+.textarea-container .textarea:disabled:hover {
+    background-color: $textarea-disabled-background-color;
+    color: $light-grey;
+}
+
+.textarea-container .textarea:hover {
+    background-color: $textarea-hover-background-color;
+    border: solid 1px $textarea-hover-border-color;
+}
+
+.textarea-container .textarea:focus {
+    background-color: $textarea-focus-background-color;
+    border: solid 2px $textarea-focus-border-color;
+    color: $textarea-font-color;
+    margin: -1px 0px;
+}
+
+.textarea-container .description-label {
+    margin-bottom: 7px;
+}
+
+.textarea-container .help-label {
+    color: $light-grey;
+    margin-top: 4px;
+}
+</style>
+
+<script>
+export const TextareaPlatforme = {
+    name: "textarea",
+    props: {
+        id: {
+            type: String,
+            default: null
+        },
+        value: {
+            type: String,
+            default: ""

```

Comment:
Use `null` for "thing that is optional and is not defined".  Empty string means a slightly different thing: "this is defined, but it happens to be empty".